demos/projects/ST/b-l4s5i-iot01a/config/demo_config.h

```diff
@@ -510,4 +510,16 @@ static unsigned char root_cert_array[] = {
  */
 #define WIFI_SECURITY_TYPE  WIFI_ECN_WPA2_PSK
 
+#ifdef democonfigENABLE_DPS_SAMPLE
+    #define democonfigCHUNK_DOWNLOAD_SIZE    256
```

Comment:
I need to go and check whether these numbers can be updated for the L475 sample now that the json buffer is gone, but does the L4plus have the same space constraints, or could these numbers potentially be bigger here as well? If I remember correctly, changing these to something too big causes a failure on build, so it's a pretty fast check!

Reply:
Yea I think I was able to get north of 2k not using DPS (possibly 4k). Let me do some fudging on the numbers and see what we can handle now.

Reply:
Bumped to 4k with or without DPS